integration/spring-data/2.5/src/main/java/com/blazebit/persistence/spring/data/impl/query/CriteriaQueryParameterBinder.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2014 - 2022 Blazebit.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.blazebit.persistence.spring.data.impl.query;
+
+import com.blazebit.persistence.spring.data.base.query.AbstractCriteriaQueryParameterBinder;
+import com.blazebit.persistence.spring.data.base.query.JpaParameters;
+import com.blazebit.persistence.spring.data.base.query.ParameterMetadataProvider;
+import com.blazebit.persistence.spring.data.repository.KeysetPageable;
+import com.blazebit.persistence.view.EntityViewManager;
+import org.springframework.data.domain.Pageable;
+
+import javax.persistence.EntityManager;
+
+/**
+ * Concrete version for Spring Data 2.x.
+ * 
+ * @author Christian Beikov
+ * @since 1.3.0
```

Comment:
You may want to replace `1.3.0` and `1.2.0` with `1.6.7` in `integration/spring-data/2.5/`, `integration/spring-data/2.6/` and `integration/spring-data/2.7/`

Reply:
Thanks, but since these files are just copies, I guess this is fine. Don't want to change stuff at this point, otherwise the checkstyle gods might be angry with me 🤣

Reply:
Oh I had my part of the checkstyle gods 😉 

Finally had plugins/configurations all set, then recently switched to M1 and now I have to go through it all again 🤣